x-pack/test/functional/apps/upgrade_assistant/feature_controls/upgrade_assistant_security.ts

```diff
@@ -9,22 +9,16 @@ import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function ({ getPageObjects, getService }: FtrProviderContext) {
-  const esArchiver = getService('esArchiver');
   const security = getService('security');
   const PageObjects = getPageObjects(['common', 'settings', 'security']);
   const appsMenu = getService('appsMenu');
   const managementMenu = getService('managementMenu');
 
   describe('security', function () {
     before(async () => {
-      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
```

Comment:
Out of curiosity, why were these changes necessary?

Reply:
I talked with @cuff-links briefly about it and we agreed it didn't seem necessary since this test suite is only changing roles. I was also occasionally getting strange SO migration errors when running locally.